### PR TITLE
Documentation on TOP parameterization in ELM

### PR DESCRIPTION
**Pull request by bishtgautam**
Adds the technical documentation for parameterization of the topographic solar radiation.

[BFB]

### CODE REVIEW

**Comment by bishtgautam** [comment]
This PR will be rebased after #5978 is merged.

**Comment by bishtgautam** [comment]
@mahf708 The bibtex plugin isn't installed (see https://github.com/E3SM-Project/E3SM/actions/runs/6473276791/job/17575629755?pr=5980#step:6:28). How can that be installed?

**Comment by bishtgautam** [comment]
@mahf708 Thank You!!! I'm not a fan of the current citation format, bu will live with it until someone finds a better solution.  

However, I can't get equation numbering to work for https://e3sm-project.github.io/E3SM/pr-preview/pr-5980/ELM/tech-guide/top_solar_parameterization/. Would you happen to know how I can fix it?

**Comment by bishtgautam** [comment]
That solution is thanks to @jeff-cohere from another project.

**Comment by bishtgautam** [comment]
@daleihao The rendered doc is https://e3sm-project.github.io/E3SM/pr-preview/pr-5980/ELM/tech-guide/top_solar_parameterization/. Could you please review this PR? I don't know why I can't add you as a reviewer for this PR.

**Comment by bishtgautam** [comment]
The equations are now getting rendered correctly when you browse the file on Github https://github.com/E3SM-Project/E3SM/blob/c96e712457177a532349f5e51e6281bc77eab68c/components/elm/docs/tech-guide/top_solar_parameterization.md. The equation numbering and cross-referencing isn't working, but I think that is okay.

**Comment by bishtgautam** [comment]
Here is the updated link to the file https://github.com/E3SM-Project/E3SM/blob/26e5834116b1a3bb9e24114ce4a32dde5273c73d/components/elm/docs/tech-guide/top_solar_parameterization.md

**Review by rljacob — APPROVED** [review]

**Comment by rljacob** [comment]
@peterdschwartz please start merging this.

**Comment by rljacob** [comment]
@peterdschwartz did you merge this to next?

**Comment by peterdschwartz** [comment]
on next as well.

File: components/elm/docs/tech-guide/index.md [hunk 1]
```diff
@@ -1 +1,3 @@
 start of the ELM Technical Guide
+
+- [TOP Parameterization](top_solar_parameterization.md): Solar Radiation ParaTopgraphic parameterization for
```
Comment (rljacob) [inline components/elm/docs/tech-guide/index.md:3]: @bishtgautam this sentence is cut off.  Will need another PR to fix it.

**Pull request merged**